app/src/main/java/com/randomappsinc/studentpicker/editing/RenameDialog.java

```diff
@@ -37,10 +36,12 @@ public RenameDialog(Context context, final Listener listener) {
                 .alwaysCallInputCallback()
                 .negativeText(android.R.string.no)
                 .positiveText(R.string.next)
+                .onNeutral((dialog, which) -> {
+                    dialog.getInputEditText().setText("" + currentMaxAmount);
```

Comment:
This feels kind of hacky, and I think it's slower than just converting it normally.
```suggestion
                    dialog.getInputEditText().setText(String.valueOf(currentMaxAmount));
```

Reply:
I put String.ValuveOf at first but then I looked at it and said "" + will make code shorter